scripts/publish.ts

```diff
@@ -0,0 +1,44 @@
+import path from 'path';
+import { readJSON, writeJSON } from './utils/json';
+import { spawn } from './utils/spawn';
+
+/** This script publishes the package as `pixi.js` and `pixijs` */
+async function main(): Promise<void>
+{
+    try
+    {
+        // Save the paths to package.json and package-lock.json
+        const packagePath = path.join(process.cwd(), 'package.json');
+        const packageLockPath = path.join(process.cwd(), 'package-lock.json');
+
+        // Read the original package.json and package-lock.json
+        const originalPackage = await readJSON<{name: string}>(packagePath);
+        const originalPackageLock = await readJSON<{name: string}>(packageLockPath);
+
+        // Run npm publish
+        await spawn('npm', ['publish']);
+
+        // Update the name in package.json and package-lock.json
+        const updatedPackage = { ...originalPackage, name: 'pixijs' };
+        // Update all instances of the package name in package-lock.json
+        const updatedPackageLock = JSON.parse(JSON.stringify(originalPackageLock)
+            .replace(new RegExp(originalPackage.name, 'g'), 'pixijs'));
+
+        await writeJSON(packagePath, updatedPackage);
+        await writeJSON(packageLockPath, updatedPackageLock);
+
+        // Run npm publish again
+        await spawn('npm', ['publish']);
+
+        // Restore the original package.json and package-lock.json
+        await writeJSON(packagePath, originalPackage);
+        await writeJSON(packageLockPath, originalPackageLock);
```

Comment:
I might suggest doing something like this to catch any errors with the publish and make sure everything in the workspace is always reverted.

```suggestion
        try {
             // Run npm publish again
             await spawn('npm', ['publish']);
        }
        finally {         
            // Restore the original package.json and package-lock.json
            await writeJSON(packagePath, originalPackage);
            await writeJSON(packageLockPath, originalPackageLock);
        }
```

Reply:
Oops just noticed the try/catch above. So, not quite this, but some way to make sure JSON is always reverted in case something goes wrong.